jax/_src/xla_bridge.py

```diff
@@ -106,6 +106,16 @@
 )
 
 
+# Warn the user if they call fork(), because it's not going to go well for them.
+def _at_fork():
+  warnings.warn(
+    "Calling os.fork() is incompatible with JAX and will likely lead to a "
+    "deadlock. os.fork() is incompatible with multithreaded code, such as JAX.",
```

Comment:
```suggestion
    "os.fork() was called. os.fork() is incompatible with multithreaded code, "
    "and JAX is multithreaded, so this will likely lead to a deadlock.",
```

Just wanted to tweak the phrasing, which seemed repetitive. Feel free to reject!

Reply:
Done. I just cut the first sentence.